test/README.md

```diff
@@ -0,0 +1,54 @@
+# Detailed test-fixture build steps
+
+The following steps will attempt to outline the current build process for the Expo test fixture contained within this repo for future reference.
+
+## Requirements
+
+The requirements for a machine building the test fixture on CLI can be found within the `platforms-ansible` repository.
+
+## Build process
+
+Depending on the build type, the build is initiated via calling the `test/scripts/build-android.sh` or `test/scripts/build-ios.sh` scripts.
+
+### Common steps
+
+Each script initially calls the `test/scripts/build-common.sh` script.  This sets up the bugsnag expo packages by:
+
+1. Clearing any `build` directory that may have been created previously
+2. Installing the `bugsnag-expo` package requirements
+3. Packaging the repository into `.tgz` installable packages and moving them into the test fixture
+
+All subsequent steps are executed within the test-fixture directory, `test/features/fixtures/test-app`
+
+4. The `bugsnag-expo-cli` package is installed from the tarball, and a script executes the following commands:
+    - `bugsnag-expo-cli set-api-key` to set a pre-defined test api key in the app configuration
+    - `bugsnag-expo-cli add-hook` to add source-map upload hooks into the app configuration
```

Comment:
the source map hook doesn't get run for an eas build, so this will need to be updated with 'add-plugin'